tensorboard/plugins/image/summary_v2.py

```diff
@@ -27,6 +27,41 @@
 def image(name, data, step=None, max_outputs=3, description=None):
     """Write an image summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
```

Comment:
Putting this "see also" at the top seems a bit odd to me, but it looks like this was discussed on #4796 so I'll comment there.

Reply:
Ok, caught up to the conversation on https://github.com/tensorflow/tensorboard/pull/4797#discussion_r597312820, so this is fine as is.